### DISCUSSION TITLE
【求助】

### DISCUSSION THREAD

**Discussion by zheloveying** [share121/inter-knot#1253] [其他]
本人在up的同意下，制作了绳网移动端登陆教程，但是B站一直锁我稿件，我也没有在视频中展示使用非法工具 申诉也不通过，各位绳匠能不能告诉我该怎么办

**Comment by sishui-buyin** [comment]
创个Q群发群里

**Comment by Jiucheng258** [comment]
问题来了，在绳网遇见超级大的麻烦怎么办？
————当然是去求助传奇绳匠“法厄同”啊！
